glue_jupyter/common/state_widgets/tests/test_viewer_image.py

```diff
@@ -0,0 +1,12 @@
+def test_contour_levels(app, data_image):
+    viewer = app.imshow(data=data_image)
+    widget_state = viewer._layout_layer_options.layers[0]['layer_panel']
+    layer_state = widget_state.layer_state
+    layer_state.level_mode = "Custom"
+
+    widget_state.c_levels_txt = '1e2, 1e3'
```

Comment:
Maybe add a test case that mix both notation up. Like `'1e-3,0.1,1,1.1,1.1e2'` or something.

Reply:
done